tracks/python/exercises/high-scores/mentoring.md

```diff
@@ -1,79 +1,62 @@
+Another variation of this solution includes the start at -1 for slicing the result to `personal_top_three`. The last solution works due the step being -1. Since no start is defined the start will default to 0 and the first step will start at -1 due to the step being defined.
 ```python
-    def personal_top_three(self):
-        return self.sorted_scores[-1:-4:-1]
+    def personal_top_three(scores):
+        return sorted(scores)[-1:-4:-1]
 ```
 
 ### Common Suggestions
 
 - Use **`sorted()`** over **`sort()`**.   `sorted()` returns a _**copy**_ , `sort()` _**mutates in place**_.  
-  Solutions that use `sort()` without first _copying_ the data in `self.scores`  erase the "latest" score.  If `latest()` is called after the `sort()`, it will be inaccurate.
--  Use built-in functions like **`max(self.scores)`** -OR- **`sorted(self.scores, reverse=True)[0]`**  over looping manually.   It's is more efficient than iterating through `self.scores` to find the largest value.
--  Using **`sorted(self.scores, reverse=True)[:3]`**  is preferable to  iterating through scores to find the three largest values.
-- Checking  **`len(self.scores) < 3`**  is unnecessary if a list slice is used to make the three highest scores list.  Python will return a copy of the entire list if it is shorter than the requested list slice.
-- A _negative index_ (e.g. **`self.scores[-1]`**) is preferred over performing a length-check  to get the index of the last element (e.g.**`self.scores[len(self.scores)-1]`**). 
-- The syntax of  **`sorted(self.scores, reverse=True)[:3]`** is preferred over  **`sorted(self.scores, reverse=True)[0:3]`**.  All slices default to zero unless otherwise specified - so convention is to leave it off.
+  Solutions that use `sort()` without first _copying_ the data in `scores`  effectively erase the "latest" score.  If `latest()` is called after the `sort()`, it will be inaccurate.
+-  Use built-in functions like **`max(scores)`** -OR- **`sorted(scores, reverse=True)[0]`**  over looping manually.   It's is more efficient than iterating through `scores` to find the largest value.
+-  Using **`sorted(scores, reverse=True)[:3]`**  is preferable to  iterating through scores to find the three largest values.
+- Checking  **`len(scores) < 3`**  is unnecessary if a list slice is used to make the three highest scores list.  Python will return a copy of the entire list if it is shorter than the requested list slice.
+- A _negative index_ (e.g. **`scores[-1]`**) is preferred over performing a length-check  to get the index of the last element (e.g.**`scores[len(scores)-1]`**). 
+- The syntax of  **`sorted(scores, reverse=True)[:3]`** is preferred over  **`sorted(scores, reverse=True)[0:3]`**.  All slices default to zero unless otherwise specified - so convention is to leave it off.
```

Comment:
```suggestion
- The syntax of **`sorted(scores, reverse=True)[:3]`** is preferred over **`sorted(scores, reverse=True)[0:3]`**. All slices default to zero unless otherwise specified - so convention is to leave it off.
```